src/main/resources/fabric-installer.json

```diff
@@ -21,14 +21,6 @@
         "name": "net.fabricmc:access-widener:1.0.0",
         "url": "https://maven.fabricmc.net/"
       },
-      {
-        "name": "net.fabricmc:fabric-loader-sat4j:2.3.5.4",
-        "url": "https://maven.fabricmc.net/"
-      },
-      {
-        "name": "com.google.jimfs:jimfs:1.2-fabric",
-        "url": "https://maven.fabricmc.net/"
-      },
```

Comment:
Does the new ow2 sat4j need to be shipped though?

Reply:
The plan is to shade it, so not relevant for the installer.